plugin.go

```diff
@@ -482,3 +499,172 @@ func prepend(prefix, s string) string {
 
 	return s
 }
+
+func GetSlackIdsOfCommitters(p *Plugin) error {
+	if p.Config.CommitterListGitPath == "" {
+		p.Config.CommitterListGitPath = os.Getenv("DRONE_WORKSPACE")
+	}
+	slackIdMap := map[string]interface{}{}
+
+	emails, err := GetChangesetAuthorsList(p.Config.CommitterListGitPath)
+	if err != nil {
+		return fmt.Errorf("failed to get git emails: %w", err)
+	}
+
+	for _, email := range emails {
+		slackId, err := getSlackUserIDByEmail(p.Config.AccessToken, email)
+		if err != nil {
+			continue
+		}
+		slackIdMap[email] = slackId
+	}
+
+	jsonStr, err := json.Marshal(slackIdMap)
+	if err != nil {
+		return fmt.Errorf("failed to marshal git emails to json: %w", err)
+	}
+	err = WriteEnvToOutputFile("COMMITTER_SLACK_IDS", string(jsonStr))
+	if err != nil {
+		return fmt.Errorf("failed to write git emails to output file: %w", err)
+	}
+	return nil
+}
+
+func GetSlackIdFromEmail(p *Plugin) error {
+
+	slackId, err := getSlackUserIDByEmail(p.Config.AccessToken, p.Config.SlackIdOf)
+	if err != nil {
+		return fmt.Errorf("failed to get Slack ID by email: %w", err)
+	}
+	err = WriteEnvToOutputFile("SLACK_ID_FOR_EMAIL", slackId)
```

Comment:
```suggestion
	err = WriteEnvToOutputFile("SLACK_ID_FROM_EMAIL", slackId)
```

even this can be a comma separated string if multiple email are passed (eg: x@h.com,y@i.com,x@j.com) and the exported variable will contain (jhghjerger,jhsvdhjfd,oibsjkdbf)